content/blog/how-to-monitor-hpe-compute-ops-management.md

```diff
@@ -0,0 +1,362 @@
+---
+title: How to monitor HPE Compute Ops Management infrastructure with Grafana
+  Metrics Dashboards
+date: 2022-10-19T18:21:24.504Z
+author: Lionel Jullien
+authorimage: /img/small-size-id.jpg
+---
+![]()
+
+<style>ul li{ font-size:26px;}</style>
+
+<style> i{ color:grey;font-family:'Courier New';font-size:22px; } </style>
+
+The purpose of this blog post is to describe how to generate Grafana dashboards to monitor any HPE Compute infrastructure managed by HPE Compute Ops Management.
+
+# Grafana Dashboards
+
+IT infrastructure metrics visualization is critical for health monitoring, prediction, and capacity planning. It provides a powerful way of viewing infrastructure utilization, revealing issues and helping maintain uninterrupted services.
+
+Grafana’s time-series graphs are the perfect enabler for IT infrastructure optimization. They can assist administrators in monitoring temperature changes, network traffic performance, power consumption, and much more. They can be used to compare data over time to note trends and detect issues, allowing administrators to make any necessary adjustments and prevent downtime.
+
+The following picture shows a typical HPE infrastructure dashboard with different panels generated from HPE Compute Ops Management:
+
+![](/img/2022-10-19-15_14_34-hpe-com-using-infinity-uql-native-api-calls-grafana-—-mozilla-firefox.png)
+
+# HPE Compute Ops Management REST API
+
+HPE Compute Ops Management provides a northbound RESTful [API ](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/)that supports many operations. All the data you can get from the HPE Compute Ops Management API can be leveraged to create beautiful and instructive Grafana dashboards and the simplest solution is to use a generic Grafana plugin that can handle REST requests, parse json responses and generate tables. With this solution, we greatly reduce the complexity of the solution which in principle requires a database like Prometheus or InfluxDB. In this post, we will see how to do without a database...
```

Comment:
Bonjour, Jullien! Wonderful to have you submit another post. Thank you. Once again, great writing.  I can tell I'll be making just a few suggestions here.
My first suggestion is here in the 2nd and 3rd line. I would break up the sentence that starts with "All the data you can get..." and currently ends with "... generate tables." into 2 sentences for ease of reading and clarity. 

The way it is currently worded, you give a nod to the fact that there are multiple solutions out there, but I'm not sure that's necessary and it complicates things a little later. I'll make some suggestions to simplify it first.

Perhaps the following would work. Stop the first sentence at "...Grafana dashboards." Then, the next sentence could read:
"To take advantage of this, simply use a generic Grafana plugin that can handle REST requests, parse json responses and generate tables."

Then, for the second to last sentence, you could change that to read
"With this solution, you can greatly reduce the complexity involved in other ways of handling it that might require a database like Prometheus or Influx DB."

Don't forget - try to avoid the "royal we" in blog posts, and instead think about you talking directly to a single reader.

The last sentence could then read "In this post, I will show you how to do it without a database..."
(you are currently missing the word "it" prior to "without", so don't forget to add that in.